lib/board.rb

```diff
@@ -0,0 +1,52 @@
+class Board
+  attr_reader :squares, :wining_combinations
+  attr_writer :squares
+
```

Comment:
## attr_accesor
Due to you're using `attr_reader` and `attr_writer` accessors use directly `attr_accessor `